flink-table/flink-table-common/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,10 @@
+flink-table
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the BSD license.
+See bundled license files for details.
+
+- com.ibm.icu:icu4j:65.1
```

Comment:
If you don't bundle this dependency into this module's jar, I don't think you need to add this file

Reply:
Same applies to LICENSE.icu4j

Reply:
Seems you need to add these to both planner's module, not here

Reply:
make sense~